projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -483,8 +483,8 @@ public function is_wpcom_atomic() {
 	 * @return bool
 	 */
 	public function is_wpcom_staging_site() {
-		if ( function_exists( 'has_blog_sticker' ) ) {
-			return has_blog_sticker( 'staging_site' );
+		if ( function_exists( 'is_blog_wpcom_staging' ) ) {
```

Comment:
It should be fine as we check for function existence. However, would it be better to follow the `get_wpcom_production_blog_id` approach and declare it as abstract on the Jetpack side, and implement it on the WPCOM side in the trait?

Reply:
@wojtekn Sure, could do in a follow-up PR.

Reply:
Created an issue for it https://github.com/Automattic/dotcom-forge/issues/2168